Sources/Kommunicate/Classes/KMConversationListViewController.swift

```diff
@@ -343,11 +343,16 @@ public class KMConversationListViewController: ALKBaseViewController, Localizabl
     }
 
     func setupBackButton() {
-        let back = LocalizedText.leftBarBackButtonText
-
-        let leftBarButtonItem = UIBarButtonItem(title: back, style: .plain, target: self, action: #selector(customBackAction))
-
-        if !configuration.hideBackButtonInConversationList {
+        guard !configuration.hideBackButtonInConversationList else { return }
+        if configuration.enableBackArrowOnConversationListScreen {
+            var backImage = UIImage(named: "icon_back", in: Bundle.kommunicate, compatibleWith: nil)
```

Comment:
Create an extension for this. We are already using it at 2 other places.

Reply:
created extensions